docs/admin.md

```diff
@@ -0,0 +1,96 @@
+
+![bs-conf](https://github.com/gazarenkov/janus-idp-operator/assets/578124/d56cbbb0-781c-43fc-8624-8832893fede3)
+ 
+
+Mapping of configMap keys (yaml files) to runtime objects (NOTE: for the time (Dec 20'23) it is a subject of change):
+| Key/File name         | k8s/OCP Kind        | Mandatory*    |ver.|Notes                                      |
+| ----------------------|:-------------------:| -------------------|------------------------------------------:|
+| deployment.yaml       | appsv1.Deployment   | Yes           |all |Backstage deployment  |
+| service.yaml          | corev1.Service      | Yes           |all |Backstage Service |
+| db-statefulset.yaml   | appsv1.Statefulset  | For DB enabled|all |PostgreSQL statefulSet    |    
+| db-service.yaml       | corev1.Service      | For DB enabled|all |PostgreSQL Service   |
+| db-service-hl.yaml    | corev1.Service      | For DB enabled|all |PostgreSQL Service   |
+| db-secret.yaml        | corev1.Secret       | For DB enabled|all |Secret to connect Backstage to PSQL   |
+| route.yaml            | openshift.Route     | No (for OCP)  |all |Route exposing Backstage service    |
+| app-config.yaml       | corev1.ConfigMap    | No            |0.02|Backstage app-config.yaml    |
+| configmap-files.yaml  | corev1.ConfigMap    | No            |0.02|Backstage config file inclusions from configMap   |
+| configmap-envs.yaml   | corev1.ConfigMap    | No            |0.02|Backstage env variables from configMap    |
+| secret-files.yaml     | corev1.Secret       | No            |0.02|Backstage config file inclusions from Secret   |
+| secret-envs.yaml      | corev1.Secret       | No            |0.02|Backstage env variables from Secret    |
+| dynamic-plugins.yaml  | corev1.ConfigMap    | No            |0.02|dynamc-plugins config *    |
+| dynamic-plugins-      | corev1.ConfigMap    | No            |0.01|dynamc-plugins config *    |
+|configmap.yaml         |                     |               |    | |
+| backend-auth-         | corev1.ConfigMap    | No            |0.01|backend auth config    |
+|configmap.yaml         |                     |               |    | |
+
+
+
+NOTES: 
+ - Mandatory means it is needed to be present in either (or both) Default and CR Raw Configuration.
+ - dynamic-plugins.yaml is a fragment of app-config.yaml provided with RHDH/Janus-IDP. The reason it mentioned separately is specific way it provided (via mounting to dedicated initContainer)  
+ - items marked as version 0.01 are not supported in version 0.02 
+### Operator Bundle configuration 
+
+With Backstage Operator's Makefile you can generate bundle descriptor using *make bundle* command
+
+Along with CSV manifest it generates default-config ConfigMap manifest, which can be modified and applied to Backstage Operator
+TODO: is kubectl sufficient or there are some other tools?
```

Comment:
you mean like `oc` ?

Reply:
I mean we need document how to Admin is able to make change in default operator configuration.
He/she may have own configuration file (based on generated one) and apply it using kubectl or oc when needed.